.github/workflows/db-demo.yml

```diff
@@ -0,0 +1,40 @@
+name: Update testops DB - demo data 
+
+on:
+  schedule:
+    - cron:  "*/30 * * * *" 
+
+jobs:
+  deploy:
+    name: Update testops DB 
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v2
+
+      - name: Setup python 
+        uses: actions/setup-python@v1
+
+      - name: Establish Cloud SQL Proxy 
+        env:
+          GCLOUD_AUTH: ${{ secrets.GCLOUD_AUTH}}
+          CLOUD_SQL_CONNECTION_NAME: ${{ secrets.CLOUD_SQL_CONNECTION_NAME}}
+        uses: mattes/gce-cloudsql-proxy-action@v1
+        with:
+          creds: ${{ secrets.GCLOUD_AUTH }}
+          instance: ${{ secrets.CLOUD_SQL_CONNECTION_NAME }}
+          port: 3307
+
+      - name: Install requirements 
+        uses: BSFishy/pip-action@v1
```

Comment:
alternative is just a run block with `pip install -r requirements.txt` as `pip` [is installed](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md) in Ubuntu VM.

Reply:
@AaronMT yea, you're right, it may not have an advantage because i assume each action you use has to spin up a docker container just for that